ui/src/app/shared/components/layout/layout.tsx

```diff
@@ -12,10 +12,15 @@ export interface LayoutProps {
     isExtension?: boolean;
 }
 
+const getBGColor = (theme: string): string => {
+    return theme === 'light' ? '#dee6eb' : '#100f0f';
+};
+
```

Comment:
This could be simplified to one line. Also can we use existing colors for the new two colors? For example, `ARGO_GRAY4_COLOR` is very similar to `#dee6eb`. Better we don't add more colors to the existing color library.

```suggestion
const getBGColor = (theme: string): string => theme === 'light' ? '#dee6eb' : '#100f0f';
```

Reply:
Solved, thanks Yi  
I have used the color codes from the existing color library - can't use the name in .tsx file as argo-ui can't be imported.